src/parsers/sanitiser.rs

```diff
@@ -1,7 +1,33 @@
 pub fn trim(line: &str) -> &str {
-    line.trim()
+    line.trim_matches(|c| c == ' ' || c == '\n')
```

Comment:
previously we stripped all whitespace (including tabs)
but... empty string chars at the end of aline look like
`'asdasda'\t\n`

so the tab char got cut